lib/src/features/user/ui/profile_screen.dart

```diff
@@ -235,43 +238,52 @@ class PerfCards extends StatelessWidget {
             height: 100,
             width: 100,
             child: PlatformCard(
-              child: Padding(
-                padding: const EdgeInsets.all(8.0),
-                child: Column(
-                  mainAxisAlignment: MainAxisAlignment.spaceAround,
-                  children: [
-                    Text(
-                      perf.shortName,
-                      style: TextStyle(color: textShade(context, 0.7)),
-                    ),
-                    Icon(perf.icon, color: textShade(context, 0.6)),
-                    Row(
-                      mainAxisAlignment: MainAxisAlignment.center,
-                      children: [
-                        Text(
-                            '${userPerf.rating}${userPerf.provisional == true || userPerf.ratingDeviation > kProvisionalDeviation ? '?' : ''}',
-                            style: kBold),
-                        const SizedBox(width: 3),
-                        if (userPerf.progression != 0) ...[
-                          Icon(
-                            userPerf.progression > 0
-                                ? LichessIcons.arrow_full_upperright
-                                : LichessIcons.arrow_full_lowerright,
-                            color: userPerf.progression > 0
-                                ? LichessColors.good
-                                : LichessColors.red,
-                            size: 12,
-                          ),
-                          Text(userPerf.progression.abs().toString(),
-                              style: TextStyle(
-                                  color: userPerf.progression > 0
-                                      ? LichessColors.good
-                                      : LichessColors.red,
-                                  fontSize: 12)),
+              child: InkWell(
```

Comment:
shouldn't the `InkWell` wrap the `PlatformCard` instead of being its child?

Reply:
I did it like that at first, because, as you say, it makes more sense. The thing is when I do that the `InkWell`'s border is around the `PlatformCard`'s border. I'm sure there is a way to fix this, but when I swapped the hierarchy the problem fixed itself, so I thought it was simpler and more readable to leave it like that.

Reply:
If you want, I can try to fix it, but I would leave it that way (not because of laziness, but instead due to code readability).

Reply:
You were right about this, I don't know how or why I thought this worked, but I tested it today and the border was not expecting as expected, sorry for my mistake.